### PR TITLE
Site admin faq

### PR DESCRIPTION
**Pull request by victoria-miltcheva**
**Description of changes**
- `docs/admin-tool-faq.md`: Duplicated + revised docs for DSS admins (from detect-secrets-discuss: [Admin] DSS Admin FAQ), https://github.ibm.com/git-defenders/detect-secrets-discuss/wiki/%5BAdmin%5D-DSS-Admin-FAQ
- `docs/detect-secrets-stream-faq.md`: Simplified headings and regenerated table of contents
-  `docs/admin-tool-faq.md`: Formatted

### CODE REVIEW

File: docs/ghe-site-admin-faq.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+## As a site admin, how do I enable/disable the pre-receive hook?
+
+First, log in to your company GitHub Enterprise instance as a site admin. Navigate to the Hooks page by clicking "Site Admin" -> "Admin Center -> "Hooks".
+
+**For a site admin to enable the pre-receive hook** (already configured):
+
+- check `Use the exit-status to accept or reject pushes`
+- uncheck `Enable this pre-receive hook on all repositories by default`
+- check `Administrators can enable and disable this hook`
```
Comment (justineyster) [inline docs/ghe-site-admin-faq.md:23]: We enable the pre-receive hook in some kind of test/debug mode that is always non-blocking, allowing the scan to complete asynchronously. We may need a site admin to help us figure out exactly what the option is called, but that's definitely worth noting here!

Comment (justineyster) [reply docs/ghe-site-admin-faq.md:23]: The more I look into this the more I think this documentation is likely out of date with how we actually have it configured. I think we probably uncheck 'Use the exit-status to accept or reject pushes' which makes the hook always non-blocking. And we may uncheck 'Administrators can enable and disable this hook'. Again though, need confirmation from a site admin. But I found this screenshot in [GitHub's documentation](https://docs.github.com/en/enterprise-server@3.1/admin/policies/enforcing-policy-with-pre-receive-hooks/managing-pre-receive-hooks-on-the-github-enterprise-server-appliance) that makes me believe that unchecking the first option results in the test/debug mode I was referring to in my first comment.
hook-name

Comment (victoria-miltcheva) [reply docs/ghe-site-admin-faq.md:23]: Note: John Edwards will verify this. Also:
>Summary - 5 seconds still applies, to the Appliance for kyndryl also.  Also we use the hook in test mode so it doesn't stop the commit with non-zero RC.

Comment (justineyster) [reply docs/ghe-site-admin-faq.md:23]: @victoria-miltcheva Were you able to confirm what settings we use?

**Review by killuazhu — CHANGES_REQUESTED** [review]
Thanks for making the change, I've added some clarifications for pre-receive hook setup.

File: docs/ghe-site-admin-faq.md (hunk 1 above)
Comment (victoria-miltcheva) [reply docs/ghe-site-admin-faq.md]: Addressed! https://github.com/IBM/detect-secrets-stream/pull/20/commits/ae8c667d975d1fc7f8531e35e2be334b42699b18

**Review by killuazhu — APPROVED** [review]
LGTM
